features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Allow checking out the correct previous Git branch after running a Git Town commmand that deletes the previous and current branches
```

Comment:
How does this sound to you:
> Feature: when the previous and current branches are deleted during a Git Town command, the main branch becomes the new previous branch"

Your suggestion suggests that deleting the branches has a direct consequence of updating the branch history, but I see it as more of a side-effect.